runestone/dragndrop/js/dragndrop.js

```diff
@@ -341,7 +341,15 @@ DragNDrop.prototype.dragEval = function (logFlag) {
     this.renderFeedback();
     if (logFlag) {  // Sometimes we don't want to log the answers--for example, on re-load of a timed exam
         let answer = this.pregnantIndexArray.join(";");
-        this.logBookEvent({"event": "dragNdrop", "act": answer, "answer":answer, "minHeight": this.minheight, "div_id": this.divid, "correct": this.correct});
+        this.logBookEvent({
+            "event": "dragNdrop",
+            "act": answer,
+            "answer": answer,
+            "minHeight": this.minheight,
+            "div_id": this.divid,
+            "correct": this.correct,
+            "dragNum": this.dragNum
```

Comment:
I'm not sure how dragNum helps us, it really only tells how many things were available to drag.  What I really want back at the server is that number divided by the number that were correct -- this.correctNum

Brad

Reply:
The idea was that with dragNum we have all the information we need to calculate the percentage on the server-side, instead of having the client preprocess the data.

Reply:
That would be fine, but I don't think we do have enough information.  Correct is a boolean. and answer only tells us the matchings.  The server doesn't know which matches were correct.

Reply:
Oops, missed that correct and correctNum are different values!